Nos/Views/ActionButton.swift

```diff
@@ -87,17 +89,11 @@ struct SecondaryActionButton: View {
     var body: some View {
         ActionButton(
             title: title,
+            font: font,
             image: image,
             imageAlignment: imageAlignment,
-            depthEffectColor: Color(hex: "#514964"),
-            backgroundGradient: LinearGradient(
-                colors: [
-                    Color(hex: "#736595"),
-                    Color(hex: "#736595")
```

Comment:
I'm not a fan of applying this new secondary color gradient everywhere. The new gradient pops too much and draws more attention to these buttons than the old color - and the point of the secondary color is that it _doesn't_ pop or draw too much attention. It also looks wrong when it appears next to the `PrimaryAction` button because one gradient is horizontal and the other vertical:
| Old | New |
|--------|--------|
| ![Screenshot 2024-08-05 at 10 22 22 AM](https://github.com/user-attachments/assets/31efea73-ccb2-43aa-b616-319db7696559) | ![Screenshot 2024-08-05 at 10 21 32 AM](https://github.com/user-attachments/assets/c894a39d-1cef-4301-ab54-4e5fa4c4d6a3) | 

I also don't think Sebastian was intending for us to use this new gradient everywhere. I'm for consistency but in this case I think using the old gradient on the "new notes" toast or just using `ActionButton` directly in `NewNotesButton` would be better.

Reply:
That makes sense. I was going for consistency and may have taken it too far. I did ask for Linda's thoughts in the ticket and it sounded to me like she [wanted](https://github.com/planetary-social/nos/issues/1070#issuecomment-2260960504) the new button style, at least for the New Notes Available button. But I didn't provide a full picture of what the old style would look like, so I could either do that, or I could make the new style _only_ apply to this new button.